### PR TITLE
Frontend setup

### PR DESCRIPTION
**Pull request by dlwong**
@poema-cavalcanti @janicelam21 @creativespoon could someone take a look? Thanks!

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ]
+}
```
Comment (creativespoon) [inline .babelrc:6]: Ok I trust that you have a newline

File: client/src/index.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
-import ReactDom from 'react-dom';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx';
 
-ReactDom.render(<div>Hello World</div>, document.getElementById('app'));
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (creativespoon) [inline client/src/index.js:6]: Would there be a future conflict if we all call our modules App?

**Review by creativespoon — APPROVED** [review]
Looks great (besides the newline warnings haunting you), approved!

File: client/src/index.js (hunk 2 above)
Comment (dlwong) [reply client/src/index.js:6]: I'm either going to change the name to photo or hash it

File: .babelrc (hunk 1 above)
Comment (dlwong) [reply .babelrc:6]: Thank you

**Pull request merged**